<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clicker</class>
 <widget class="QMainWindow" name="Clicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clicker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>151</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Clicker1">
       <property name="text">
        <string>Auto Clicker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clicker2">
       <property name="text">
        <string>Super Clicker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clicker3">
       <property name="text">
        <string>Mega Clicker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clicker4">
       <property name="text">
        <string>Quantum Clicker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clicker5">
       <property name="text">
        <string>Lateralus</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>61</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="Clicker1Count">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker2Count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker3Count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker4Count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker5Count">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>101</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="Clicker1Cost">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker2Cost">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker3Cost">
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker4Cost">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Clicker5Cost">
       <property name="text">
        <string>5000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>171</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="c1u1">
       <property name="text">
        <string>Auto Clicker Enhancer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c1u2">
       <property name="text">
        <string>Better Auto Clicker Circuts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c1u3">
       <property name="text">
        <string>Well Oiled Auto Clickers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c2u1">
       <property name="text">
        <string>Super Clicker Enhancer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c2u2">
       <property name="text">
        <string>Polished Super Clickers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c2u3">
       <property name="text">
        <string>Luxurious Super Clickers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c3u1">
       <property name="text">
        <string>Mega Clicker Enhancer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c3u2">
       <property name="text">
        <string>Reinforced Mega Clickers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c3u3">
       <property name="text">
        <string>Carbon Fibre Mega Clickers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c4u1">
       <property name="text">
        <string>Quantum Clicker Enhancer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c4u2">
       <property name="text">
        <string>Energized Quantum Clcikers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c4u3">
       <property name="text">
        <string>Bose-Einstein Quantum Clickers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c5u1">
       <property name="text">
        <string>Spiral Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c5u2">
       <property name="text">
        <string>Keep Going</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c5u3">
       <property name="text">
        <string>The Holy Gift</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ClickerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Click</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777207</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DisplayTitle">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clicks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpsTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clicks Per Second</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpsDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReset">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuReset"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
